Current master panics with "inconsistent label cardinality"

From one terminal, ping an unreachable target:

```
$ sudo ./ping_exporter 10.255.255.254
```

From another terminal, continuously request metrics:

```
$ while true; do curl -s localhost:9427/metrics; done
```

The panic occurs before any metrics are returned, it seems to happen during the request which would have returned metrics:

```
INFO[0000] adding target for host 10.255.255.254 (10.255.255.254)  source="target.go:72"
INFO[0000] Starting ping exporter (Version: 0.4.3)       source="main.go:147"
INFO[0000] Listening for /metrics on :9427               source="main.go:167"
panic: inconsistent label cardinality

goroutine 316 [running]:
github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus.MustNewConstMetric(0xc420193d50, 0x2, 0x3ff0000000000000, 0xc420133b60, 0x2, 0x2, 0x2, 0x2)
	/home/nicholash/Src/go/src/github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus/value.go:172 +0xb0
main.(*pingCollector).Collect(0xc4200a1640, 0xc420146f60)
	/home/nicholash/Src/go/src/github.com/czerwonk/ping_exporter/collector.go:70 +0x179
github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc4200b5580, 0xc420146f60, 0x92ee60, 0xc4200a1640)
	/home/nicholash/Src/go/src/github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:433 +0x61
created by github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/home/nicholash/Src/go/src/github.com/czerwonk/ping_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:431 +0x302

```

--- a/collector.go
+++ b/collector.go
@@ -50,7 +50,7 @@ func (p *pingCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 
 	for target, metrics := range p.metrics {
-		l := strings.SplitN(target, " ", 2)
+		l := strings.SplitN(target, " ", 3)
 
 		if metrics.PacketsSent > metrics.PacketsLost {
 			ch <- prometheus.MustNewConstMetric(rttDesc, prometheus.GaugeValue, float64(metrics.Best), append(l, "best")...)

